plugins/ltac/rewrite.ml

```diff
@@ -1795,8 +1795,7 @@ let declare_an_instance n s args =
 let declare_instance a aeq n s = declare_an_instance n s [a;aeq]
 
 let anew_instance atts binders (name,t) fields =
-  let program_mode = atts.program in
-  let _id = Classes.new_instance ~program_mode atts.polymorphic
+  let _id = Classes.new_instance atts.polymorphic
```

Comment:
Doesn't this (and the other one in this file) change behaviour in tricky elaboration related ways?
In fact AFAICT the program attribute isn't used anymore by rewrite so if we're OK with ignoring it we should remove it from the rewrite_attributes record.

Reply:
done